koku/masu/processor/tasks.py

```diff
@@ -1266,3 +1274,43 @@ def validate_daily_data(schema, start_date, end_date, provider_uuid, ocp_on_clou
         data_validator.check_data_integrity()
     else:
         LOG.info(log_json(msg="skipping validation, disabled for schema", context=context))
+
+
+@celery_app.task(name="masu.processor.tasks.process_openshift_on_cloud_trino", queue=DownloadQueue.DEFAULT, bind=True)
```

Comment:
Reversed question from above :smile: Download VS Summary queue, just a question on which we want to use and if the current code is intentional?

Reply:
Definitely unintentional to use both here! lets use the other thread to figure out which one we want it to land in and consolidate.

Reply:
updated this to be the summary queue and match the above.